File: app/pizza_party/src/models/menu/Pizza.ts
import PizzaType from "./PizzaType";

type Pizza = {
  id: number;
  name: string | undefined;
  description: string | undefined;
  price: number;
  imageLink: string | undefined;
  listOfIngredients: string[];
  type: PizzaType;
};

export default Pizza;

File: srv/pizza_party_api/Models/Pizza.cs
namespace pizza_party_api.Models;

public class Pizza
{
    public int Id { get; set; }
    public string Name { get; set; }
    public decimal Price { get; set; }
    public virtual ICollection<CartItem> CartItems { get; set; }
    public string ImageUrl { get; set; }
    public string Description { get; set; }
    public string[] listOfIngredients { get; set; }
    public TypeOfPizza TypeOfPizza { get; set; }
}


